libraries/botframework-connector/botframework/connector/auth/microsoft_app_credentials.py

```diff
@@ -89,7 +88,7 @@ def signed_session(self) -> requests.Session:
             del session.headers['Authorization']
         return session
 
-    async def get_access_token(self, force_refresh: bool=False) -> str:
+    def get_access_token(self, force_refresh: bool=False) -> str:
```

Comment:
This looks like it's async in [C#](https://github.com/microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Connector/Authentication/MicrosoftAppCredentials.cs#L221) and [JS](https://github.com/microsoft/botbuilder-js/blob/master/libraries/botframework-connector/src/auth/microsoftAppCredentials.ts#L102) why is this sync now?

Reply:
Per discussion of msrest client not supporting async methods in their Authentication class.